src/views/includes/cookie-manager/cookie-banner.njk

```diff
@@ -0,0 +1,21 @@
+<div class="govuk-cookie-banner" id="cm_cookie_notification" aria-label="Cookies on find an MOT test centre for a heavy goods vehicle (HGV), bus or trailer">
+    <div class="govuk-cookie-banner__message govuk-width-container">
+
+        <div class="govuk-grid-row ">
+            <div class="govuk-grid-column-two-thirds">
+                <h2 class="govuk-cookie-banner__heading govuk-heading-m">Tell us whether you accept cookies</h2>
+                <div class="govuk-cookie-banner__content">
+                    <p>We use <a class="govuk-link" href="/cookies">cookies to collect information</a> about how you use the {{ serviceName }} service. We use this information to make the service work as well as possible.</p>
+                </div>
+            </div>
+        </div>
+
+        <div class="govuk-button-group">
+            <div class="js-show">
+                <button class="govuk-button" type="submit" id="cookie-accept-submit">Accept all cookies</button>
+            </div>
+            <a role="button" class="govuk-button" href="/cookie-preferences">Set cookie preferences</a>
```

Comment:
Might be worth checking the grammar on the line
'Cookies on find' - 'Cookies on finding'

Also, are the button and link here tabbable? Not sure if thats an accessibility requirement on this project or not, but sometimes links need a tabindex property to be tabbable